### PR TITLE
docs: Guide on approving cluster repave operations

### PR DESCRIPTION
**Pull request by yuliiiah**
## Describe the Change

This PR adds a guide on approving cluster repave operations.

## Review Changes

💻 Preview Links:
- [Update a Cluster](https://deploy-preview-2128--docs-spectrocloud.netlify.app/clusters/cluster-management/cluster-updates/) – Added a note after the first paragraph.
- [Node Pools > Repave Behavior and Configuration](https://deploy-preview-2128--docs-spectrocloud.netlify.app/clusters/cluster-management/node-pool/#repave-behavior-and-configuration) – Added a note at the end of the section.
- [Node Pools > Approve Cluster Repave](https://deploy-preview-2128--docs-spectrocloud.netlify.app/clusters/cluster-management/node-pool/#approve-cluster-repave) – New section.

🎫 [PEM-4282](https://spectrocloud.atlassian.net/browse/PEM-4282)

### CODE REVIEW

File: docs/docs-content/clusters/cluster-management/node-pool.md [hunk 1]
```diff
@@ -176,3 +179,43 @@ After you have modified a new node pool, you can validate the node pool by follo
 
 5. Ensure the new node pool is listed in the **Node Pools** section and that all compute instances are in the healthy
    status.
+
+## Approve Cluster Repave
+
+### Prerequisites
+
+- A Palette-deployed cluster.
```
Comment (lennessyy) [inline docs/docs-content/clusters/cluster-management/node-pool.md:187]: We usually say "An active cluster in Palette"

Comment (lennessyy) [inline docs/docs-content/clusters/cluster-management/node-pool.md:187]:
```suggestion
- An active cluster in Palette.
```

File: docs/docs-content/clusters/cluster-management/node-pool.md [hunk 2]
```diff
@@ -176,3 +179,43 @@ After you have modified a new node pool, you can validate the node pool by follo
...
+
+- Sufficient permissions to edit the cluster.
```
Comment (lennessyy) [inline docs/docs-content/clusters/cluster-management/node-pool.md:189]: Maybe we can be a little more specific? I'll leave this up to you.

**Review by lennessyy — APPROVED** [review]

File: docs/docs-content/clusters/cluster-management/node-pool.md [hunk 3]
```diff
@@ -40,10 +40,17 @@ Different types of repaving operations may occur, depending on what causes them:
   Kubernetes layer impact all nodes, such as when upgrading to a different Kubernetes version. All nodes across all
   pools are sequentially repaved starting with the control plane.
 
-You can customize the repave time interval for all node pools except the control plane pool. The default repave time
-interval is 0 seconds. You can adjust the node repave time interval during or after cluster creation. If you need to
-modify the repave time interval post-cluster creation, follow the [Change a Node Pool](#change-a-node-pool) instructions
-below.
+You can customize the repave time interval for all node pools except the control plane node pool. The default repave
+time interval is 0 seconds. You can adjust the node repave time interval during or after cluster creation. If you need
```
[the remark this answers is not in the document]
Comment (yuliiiah) [reply docs/docs-content/clusters/cluster-management/node-pool.md:44]: Good catch Lenny, thank you! I'll clarify this with Deepak this week :)

**Pull request merged**